WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInActivity.java

```diff
@@ -195,12 +213,16 @@ public SmartLockHelper getSmartLockHelper() {
     }
 
     private void popBackStackToSignInFragment() {
-        FragmentManager fragmentManager = getSupportFragmentManager();
-        while (fragmentManager.getBackStackEntryCount() > 1) {
-            fragmentManager.popBackStackImmediate();
-        }
 
-        getSupportFragmentManager().popBackStack();
+        // this can bee called asynchronously from Volley onError handler so we must check if state is OK
```

Comment:
Even though they both run on the main thread, it could happen that the activity's `onStop()` runs first and then the REST client invokes the error handler, couldn't it? Also since the fragment's `onDetach()` event is called after `onStop()`, could we not still get this exception with the right timing?